lib/helper/WebDriver.js

```diff
@@ -2722,9 +2722,9 @@ async function proceedSeeField(assertType, field, value) {
     }
   };
 
-  const proceedSingle = el => this.browser.getElementAttribute(getElementId(el), 'value').then((res) => {
+  const proceedSingle = el => this.browser.getElementProperty(getElementId(el), 'value').then((res) => {
```

Comment:
just a guess, maybe tests are failing here, because there is no check for this.browser.isW3C?

Reply:
@DavertMik  add check isW3C, thanks